docs/source/features/parallelisms.rst

```diff
@@ -3,17 +3,82 @@
 Parallelisms
 ------------
 
-NeMo Megatron supports 5 types of parallelisms (which can be mixed together arbitrarily):
+NeMo Megatron supports five types of parallelisms (which can be mixed together arbitrarily):
```

Comment:
Change to: NeMo Megatron supports five types of parallelisms (which can be mixed together arbitrarily).

Please delete colon and replace with a period